build.py

```diff
@@ -72,8 +72,8 @@
 TRITON_VERSION_MAP = {
     "2.44.0dev": (
         "24.03dev",  # triton container
-        "24.01",  # upstream container
-        "1.16.3",  # ORT
+        "24.02",  # upstream container
+        "1.17.2",  # ORT
         "2023.3.0",  # ORT OpenVINO
         "2023.3.0",  # Standalone OpenVINO
```

Comment:
have we tested that ORT works?

Reply:
Yes, pipeline with this change is in review